terraform/common.tf

```diff
@@ -87,7 +92,25 @@ variable "ca" {
 
 variable "eco_config_file" {
   description = "Defines the content of the eco config file, if not empty, then will use this config file directly instead of of the config file templates (optional)"
-  default = ""
+  default     = ""
+}
+
+variable "telegraf_config_file" {
+  description = "Defines the content of the Telegraf config file, if not empty, then it will use this config file (optional)"
+  default     = ""
+}
+
+// A telegraf outputs config
+// e.g.
+/*
+ [outputs.graphite]
+ graphite_tag_support = true
+ prefix = "telegraf"
+ servers = ["example.com:2003"]
+*/
+variable "telegraf_outputs_config" {
+  description = "Defines output(s) to append to default Telegraf config"
+  default     = "[[outputs.discard]]"
```

Comment:
last one.. sorry, should we add this var to the readme https://github.com/Quentin-M/etcd-cloud-operator/blob/master/terraform/platforms/aws/README.md? 🤷  otherwise, let's merge :)

Reply:
Yep, sorry missed this comment I'll update the readme.